docker/scripts/install_drivers.sh

```diff
@@ -0,0 +1,15 @@
+# Install RealSense
```

Comment:
To avoid unexpected behavior, it's safer to start all shell scripts with a shebang.
When using the scripts in Dockerfiles (or in other automated pipelines like CI), you probably also want errors that happened when executing the script to be propagated to the higher-level script that called it (piping failures). You can get both by starting each script with:
```
#!/bin/bash

set -o pipefail
```